extensions/azurehybridtoolkit/notebooks/hybridbook/content/appendices.md

```diff
@@ -5,7 +5,7 @@
 See the <a href="https://azure.microsoft.com/en-us/global-infrastructure/locations/">Azure locations</a> page for a complete list of Azure regions along with their general physical location. The following is a list of common North American location settings for this guide:
 
 ### Regions
-| Setting | Location |
+| Setting | Location | 
```

Comment:
Remove the space here using notepad.

Reply:
removing file. the changes were migrated to a notebook in another PR